share/smack/top.py

```diff
@@ -420,13 +420,21 @@ def verify_bpl(args):
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
 
+  elif args.verifier == 'symbooglix':
+    command = ['symbooglix']
+    command += [args.bpl_file]
+    command += ["--file-logging=0"]
+    command += ["--entry-points=%s" % ",".join(args.entry_points)]
+    command += ["--timeout=%d" % args.time_limit]
+    command += ["--max-depth=%d" % args.unroll]
```

Comment:
@zvonimir How would want nested loops to be treated? Effectively what I'm thinking of doing is having Symbooglix keep track of what loops it is in whilst executing and incrementing a per path loop head counter every time the loop head basic block is visited and resetting the counter every time that loop is escaped. If the visit count for any basic block for the head of a loop exceeds the bound then the path will be terminated. For nested loops this means the outer and inner loop are treated completely independently. Is this the behaviour you expect?